src/main/java/com/arextest/replay/schedule/sender/impl/DefaultHttpServletReplaySender.java

```diff
@@ -123,6 +123,9 @@ public boolean send(ReplayActionCaseItem caseItem) {
         before(caseItem.getRecordId());
         headers.remove(CommonConstant.AREX_REPLAY_WARM_UP);
         headers.put(CommonConstant.AREX_RECORD_ID, caseItem.getRecordId());
+        if (caseItem.getParent() != null && StringUtils.isNotEmpty(caseItem.getParent().getExclusionOperationConfig())) {
```

Comment:
use a local var for `caseItem.getParent().getExclusionOperationConfig())`

Reply:
done